addons/test_website/tests/test_controller_args.py

```diff
@@ -29,3 +33,19 @@ def test_crawl_args(self):
         req = self.url_open('/ignore_args/kw?a=valueA&b=valueB')
         self.assertEqual(req.status_code, 200)
         self.assertEqual(req.json(), {'a': 'valueA', 'kw': {'b': 'valueB'}})
+
+    def test_route_type(self):
+        HOST = 'http://127.0.0.1'
```

Comment:
Why don't you use `self.url_open` like other tests?

Reply:
@awa-odoo Because I can only perform `GET` requests with "url_open" (and bellow I need to perform `POST` requests)